From b1db3f3b002161e8955fa7b1c312cb015be17ff2 Mon Sep 17 00:00:00 2001
From: lyz <lyz@rock-chips.com>
Date: Fri, 11 Jul 2014 17:42:24 +0800
Subject: [PATCH] usb-uart: fix compiler warning

arch/arm/mach-rockchip/rk3036.c: In function 'usb_uart_init':
arch/arm/mach-rockchip/rk3036.c:87:9: warning: unused variable
---
 arch/arm/mach-rockchip/rk3036.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/arch/arm/mach-rockchip/rk3036.c b/arch/arm/mach-rockchip/rk3036.c
index 65a3d8d552b1..72317f89e002 100755
--- a/arch/arm/mach-rockchip/rk3036.c
+++ b/arch/arm/mach-rockchip/rk3036.c
@@ -84,7 +84,9 @@ static void __init rk3036_boot_mode_init(void)
 
 static void usb_uart_init(void)
 {
+#ifdef CONFIG_RK_USB_UART
     u32 soc_status0 = readl_relaxed(RK_GRF_VIRT + RK3036_GRF_SOC_STATUS0);
+#endif
     writel_relaxed(0x34000000, RK_GRF_VIRT + RK3036_GRF_UOC1_CON4);
 #ifdef CONFIG_RK_USB_UART
     if (!(soc_status0 & (1 << 14)) && (soc_status0 & (1 << 17))) {
-- 
2.35.3

